documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,53 @@
+== External Trigger
+=== IMX296
+The global shutter camera can be triggered externally by pulsing the XTR pin on the board.
+Multiple cameras can be connected to the same pulse leading to a better way to synchronise two cameras.
+
+The low puslewidth is equal to the exposure time. i.e. a low pulse of 10ms leads to an exposure time of 10ms.
```

Comment:
Maybe mention how to control the framerate as well?  Perhaps a diagram might be helpful here.